extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -75,11 +75,9 @@ require([
 		// Ads
 		adTracker.measureTime('adengine.init', 'queue.desktop').track();
 		win.adslots2 = win.adslots2 || [];
-		adEngine.run(adConfigDesktop, win.adslots2, 'queue.desktop');
-
 		scrollHandler.init();
+		adEngine.run(adConfigDesktop, win.adslots2, 'queue.desktop');
```

Comment:
@nandy-andy https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/gpt/helper.js#L67
We need to call `init` before `getReloadedViewCount`.
However, we could do it in different way, I am up for your suggestions :wink: